uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -16,63 +16,12 @@ public function install() {
         $this->{$methodName}();
       }
     }
-
-    $this->setComponentStatuses(array(
-      'CiviTask' => TRUE,
-      'CiviDocument' => TRUE,
-    ));
   }
 
   public function enable() {
-    $this->setComponentStatuses(array(
-      'CiviTask' => TRUE,
-      'CiviDocument' => TRUE,
-    ));
-    return TRUE;
   }
 
   public function disable() {
```

Comment:
Looks like the `enable` and `disable` methods are empty now. Can't we remove them?